calculatorV2.rb

```diff
@@ -0,0 +1,120 @@
+# Calculator program
+
+# Get operation input from user
+puts "Hey-oh! What kind of calculation would you like to perform?"
+operation = gets.chomp
+
+until operation == "^" || operation == "exp" || operation == "exponent" || operation == "%" || operation == "mod" || operation == "modulo" || operation == "*" || operation == "mult" || operation == "multiply" || operation == "/" || operation == "div" || operation == "divide" || operation == "-" || operation == "sub" || operation == "subtract" || operation == "+" || operation == "add" || operation == "addition"
+	puts "I\'m sorry, I don't understand. Please try again"
+	#kick it back to retrieve new operation and reassign operation
+	operation = gets.chomp
+end 	
+
+
+
+
+# Get first number input from user
+puts "Awesome. What is the first number you would like to perform that on?"
+first_number = gets.chomp
+
+if first_number == "0"
+	first_number_int = first_number.to_i
+
+	elsif first_number.to_i == 0
+		
```

Comment:
Line 23 is a little confusing.
is this another branching path off of your original if statement? If so, I recommend lining it up with the original if statement.
or is this a new nested condition after it makes it past the first if statement? If so, then it is actually a new if statement and not an elsif.